ihatemoney/models.py

```diff
@@ -264,6 +265,34 @@ def get_pretty_bills(self, export_format="json"):
             )
         return pretty_bills
 
+    def switch_currency(self, new_currency):
+        # Update converted currency
+        if new_currency != self.default_currency:
+            for bill in self.get_bills():
+                if bill.original_currency == new_currency:
```

Comment:
I'm not sure this is what we want here. 🤔 

Consider the case where we've got a project with default currency set to USD. We've got three bills : 

Amount | original_currency | converted_amount
--------- | ------------------- | ---------------------
10 | EUR | 11.88
10 | GBP | 13.79
10 | USD | 10

And now we want to convert to EUR. And the expected result is : 

Amount | original_currency | converted_amount
--------- | ------------------- | ---------------------
10 | EUR | 10
10 | GBP | 11.6
10 | USD | 8.4

Here on the first line, even if `bill.original_currency = new_currency` we need to set `bill.converted_amount = bill.amount`